### ISSUE TITLE
"/book unlock" delete all content from books :/ [CMI 9.7.5.3 & CMILib: 1.5.1.1]

### ISSUE THREAD

**Issue by Chevels** [Zrips/CMI#9056]
### Description of Issue

As soon as you use `/book unlock` it permanently destroys all the content of the books.
This affects all books:
- books written and signed a long time ago (_before 1.20.5... and even several years_)
- books written and signed recently, after 1.20.5 or 1.21 (_I did the test again this night by creating a book on purpose_)

While searching I found an [existing and supposedly fixed issue](https://github.com/Zrips/CMI/issues/8719), but it is closed and since it is a bit old I reopen another one since the problem seems to have returned.

### Version Information

```shell
CMI:
[01:05:57 INFO]: CMI: 9.7.5.3 SqLite
[01:05:57 INFO]: CMILib: 1.5.1.1 
[01:05:57 INFO]: Server: Paper(124) 1.21-R0.1-SNAPSHOT+
[01:05:57 INFO]: CMI economy: Disabled Vault: 1.7.3-CMI CMI Chat: Enabled 
[01:05:57 INFO]: Modules -> 61 enabled 5 disabled: ranks, votifier, moneyCheque, jail, kits

Server software:
[01:06:39 INFO]: This server is running Paper version 1.21-124-master@df3b654 (2024-08-02T00:02:38Z) (Implementing API version 1.21-R0.1-SNAPSHOT)
```

### Errors

```shell
There is no feedback from the command, no error, everything seems normal until you open the book
```

### Relevant Config Sections

```yaml
Nothing, I think (?)
```

### Relevant Plugins

I don't really know what to put as a relevant plugin in this section

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by Zrips** [comment]
Are you sure that this is being handled by CMI? As there isint such sub command as unsign. Did you had in mind `/cmi book unlock`?

**Comment by Chevels** [comment]
> Are you sure that this is being handled by CMI? As there isint such sub command as unsign. Did you had in mind `/cmi book unlock`?

Oh sorry, the command is indeed `/cmi book unlock`, I just checked. Since the time I've been using this command for my RP I don't know why my memory transformed it into unsign...

**Comment by Chevels** [comment]
@Zrips Hello !
any news about this issue ? :/

**Comment by Chevels** [comment]
This is weird because it affects all books in my server and our books are all normal books written by players or without formatting, commands, clickable text or anything. Nothing special. Sometimes they have a custom name or lore, but issue occur on classic books too...

I already did the test, as noted in my first post I tested on purpose to create the issue:
> This affects all books:

> - books written and signed a long time ago (before 1.20.5... and even several years)
> - books written and signed recently, after 1.20.5 or 1.21 (I did the test again this night by creating a book on purpose)

It annoys me that you can't reproduce it, we miss this feature a lot :/
The best I can do would be to show you on my server, but I suspect you don't have time for this kind of thing, which I completely understand

**Comment by Chevels** [comment]
> Retested this multiple times, still can't reproduce this issue. Are your books special ones? Like containing clickable text or similar. Does this issue persist if you sign generic new book and try to unlock it?

Hello!
So it was indeed a false alarm and I am absolutely sorry to have made you waste time with that. While doing some tests this night I discovered that another plugin on my server actually offers the `/book unsign` command (_I didn't know that essentials had this command, I thought it was a CMI command..._) and, being too confident I didn't always put the `/cmi` prefix in front of the "book" and in these cases the autocompletion offered me "`unsign`" instead of "`unlock`".

In short, the CMI command `/cmi book unlock` works perfectly. Really sorry, I got completely confused with these 2 similar commands. I hope I didn't waste your time too much :/

**Comment by Zrips** [comment]
No worriers, now we know what was the cause of this issue for future reference.
